release-tools/src/main/java/org/springframework/data/release/cli/IssueTrackerCommands.java

```diff
@@ -36,12 +38,14 @@
 
 /**
  * @author Oliver Gierke
+ * @author Mark Paluch
  */
 @CliComponent
 public class IssueTrackerCommands implements CommandMarker {
 
 	private final PluginRegistry<IssueTracker, Project> tracker;
 	private final JiraConnector jira;
+	private final GitHubIssueConnector gitHub;
```

Comment:
Not sure what you mean. All methods you introduced, take a `TrainIteration`, which contains all `ModuleIterations`, for which in turn you can obtain the `IssueTracker` instance in a generic way. That means, we don't have to create dedicated commands for Git and JIRA but can generically refer to `tracker`, just like the `tracker changelog` command already does. Which tracker is being used should be an implementation detail that shouldn't be revealed on the command level, so that a shell user doesn't have to even know there are different ones.